cmd/skupper/skupper_link.go

```diff
@@ -119,7 +119,8 @@ func NewCmdLinkStatus(skupperClient SkupperLinkClient) *cobra.Command {
 
 			linkHandler := skupperClient.LinkHandler()
 			if linkHandler == nil {
-				return fmt.Errorf("unable to retrieve links")
+				fmt.Println("Unable to retrieve links")
```

Comment:
I note the message here is slightly different, but that is due to the SkupperLinkClient interface's LinkHandler() method not providing any way to signal errors. That seems like something for a separate change.